src/components/Button/styles/standardThemes.ts

```diff
@@ -155,3 +155,32 @@ export const translucent = css`
         fill: ${color.white};
     }
 `;
+
+export const text = css`
+    background-color: ${color.white};
+    border-color: transparent;
+
+    color: ${color.blue};
+
+    :hover {
+        color: ${color.paleCerulean};
+        background-color: ${color.blueLight};
+        border-color: transparent;
+
+        svg {
+            fill: ${color.paleCerulean};
+        }
+    }
+
+    :active {
```

Comment:
Following the style in Figma, we need to have a "gray" bg in active state

![image](https://user-images.githubusercontent.com/78314301/163412486-0404eac8-c7c3-470f-8174-d5e2d6976d29.png)